apread/entries.py

```diff
@@ -114,12 +179,34 @@ def readData(self):
         # if something was wrong previously, nothing will happen here
         if self.broken:
             return
-
-        # initialize data
-        self.data = []
-        # read all channel data
-        for i in tqdm(range(self.length), leave=False):
-            self.data.append(self.reader.read_double())
+        
+        if self.fastload:
+            # The data is stored channelwise. We therefore only need to pass pointers to the first and last byte.
+            if self.precision == 8 or self.precision == 4:
+                self.data = np.fromfile(self.reader.buf, dtype=np.dtype('f{}'.format(self.precision)), count=self.length)
+            elif self.precision == 2:
+                MinValue = self.reader.read_double()
+                MaxValue = self.reader.read_double()
+                sf = (MaxValue - MinValue)/32767 # scale factor
```

Comment:
![image](https://user-images.githubusercontent.com/90382345/165456632-ded47ea9-38f7-4534-912b-749ace96e7ec.png)
Hi! In 2-byte mode, the measurement values are stored as 16-bit integers, which effectively constrains them to take on one of 2^16 different values. The parameters 'MaxValue' and 'MinValue' are the largest and smallest measured values encountered in a given channel in a measurement file.
As an example, if a velocity signal in a given measurement file varied on the interval [0; 10,3] m/s, that maximum value would be stored as the integer 32768. If you multiply the integer by the scale factor 'sf', you would then retrieve the measured value in Python.

I'll add a commit where I upload the official format specification to the repo (see screenshot above). HBK were okay with me uploading the sample measurement files to Github, so I'm sure they wouldn't mind.

Reply:
About using 'np.fromfile' to load files by default, I've been using it for a while now on my measurement files without any unexpected behaviour, so I think it would be safe to make that change.